python/cugraph/cugraph/dask/common/part_utils.py

```diff
@@ -116,6 +116,10 @@ def persist_dask_df_equal_parts_per_worker(
 
     ddf_keys = dask_df.to_delayed()
     workers = client.scheduler_info()["workers"].keys()
```

Comment:
Is this line still needed?  It looks like `workers` from this call isn't used anymore and gets overwritten by the new code on line 121.